### ISSUE TITLE
Handle image groups which are not present

### ISSUE THREAD

**Issue by jimk-bdrc** [buda-base/volume-manifest-builder#23]
In W8LS66822, 
the get_volume_infos_from_S3 queries BUDA, and gets back:
```
"item","list","grId"
"bdr:I8LS66822","","I8LS68086"
"bdr:I8LS66822","","I8LS68087"
"bdr:I8LS66822","","I8LS68088"
"bdr:I8LS66822","I8LS680890001.tif:2|I8LS680890003.jpg:85","I8LS68089"
"bdr:I8LS66822","I8LS680900001.tif:2|I8LS680900003.jpg:81","I8LS68090"
"bdr:I8LS66822","","I8LS68091"
"bdr:I8LS66822","","I8LS68092"
...
```
and chokes when there are no images in the groups. It should ignore.

**Comment by eroux** [comment]
we need to discuss that a little bit... in the next iteration of BUDA the image list will disappear so the second column will always be empty. I can change the query to include the totalImages from the xml files, but in the end it will disappear too... let me think of a good system and I'll make a proposal

**Comment by jimk-bdrc** [comment]
Hi Élie, I think there's no need. v-m-t already natively handles the case of no data in the middle column. (see branch volume-manifest-tool-20 v_m_t/VolumeInfoBuda.py:fetch (lines 74ff) if `expand_image_list(l)` returns an empty list (when the image list is empty, it then tries the next strategy `self.get_image_names_from_S3(work_rid,g)`, to go out and look for the BOM on S3. This was the step which had an unhandled exception, since there was no S3 object. The fix is to wrap that exception in a warning, which is what we will do now.

Of course, it would be ideal to have the BOM be a top level resource somewhere in BUDA, and I know we were thinking about doing something like that. When you're ready to discuss how you want to implement it, give me a shout.  My first thought is that instead of writing the BOM to the imagegroup, we use some API to deposit it to a different resource. We could also alter v-m-t to write the dimensions.json to a similar resource, to align it with the rest of volume-manifest work.

**Comment by jimk-bdrc** [comment]
fixed in #6a1b9d8, version 1.0.1b1-2020.-04.15.01 (build number)
